### PR TITLE
Hydra

### PR DESCRIPTION
**Pull request by asafpr**
Added hydra-multi recipe
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by asafpr** [comment]
@BiocondaBot please fetch artifacts

**Comment by asafpr** [comment]
@BiocondaBot please fetch artifacts

**Comment by asafpr** [comment]
@BiocondaBot please fetch artifacts

**Comment by asafpr** [comment]
@BiocondaBot please fetch artifacts

**Comment by asafpr** [comment]
@BiocondaBot please add label

File: recipes/hydra-multi/build.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+mkdir -p $PREFIX/bin
+make all CXX="${CXX}" \
+    ZLIB_PATH="${PREFIX/lib}" \
+    CXXFLAGS="${CXXFLAGS}" \
+    LIBS="${LDFLAGS} -lm -lz -lpthread"
+#    CC="${CC}" \
+#    CXX="${CXX}" \
+#    CPPFLAGS="${CPPFLAGS}" \
+#    CFLAGS="${CFLAGS}" \
+#    LDFLAGS="${LDFLAGS} -lz" \
```
Comment (natir) [inline recipes/hydra-multi/build.sh:13]: Please remove this comment line

File: recipes/hydra-multi/build.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+chmod 755 hydra-multi.sh
+chmod 755 scripts/hydraToBreakpoint.py
+sed 's[/usr/local/bin/python2.6[/usr/bin/env python[' -i scripts/frequency.py
```
Comment (natir) [inline recipes/hydra-multi/build.sh:17]: Did you create an issue or pull request ton integrate this sed in upstream ?

**Review by natir — CHANGES_REQUESTED** [review]
Thanks for your work.

I have some remarq to get a cleaner recipe.

Can you squash all your commit in one with `git rebase` and mad a `git push --force on your branch`

File: recipes/hydra-multi/build.sh (hunk 2 above)
Comment (asafpr) [reply recipes/hydra-multi/build.sh:17]: This package is not maintained for a very long time, I opened an issue for creating a release to no avail.

File: recipes/hydra-multi/build.sh (hunk 1 above)
Comment (asafpr) [reply recipes/hydra-multi/build.sh]: Pushed

**Comment by asafpr** [comment]
I'm afraid I'll screw things up if I try to squash, can't it be done when merging?

**Comment by natir** [comment]
Isn't very important in fact

**Comment by asafpr** [comment]
Ok, I'll give it a shot

**Comment by asafpr** [comment]
Damn it, I pulled from master and now new commits are on this branch too

**Comment by natir** [comment]
Good job @asafpr.

I have another concern this package has many script and binary, but many of them seem not be launch by the end-user, but only by `hydra-multi.sh`. 

@bioconda/core is it ok to leave all these script and binary in $PREFIX/bin or better store them in $PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/ and change $PATH or calling path in launch script ?

**Comment by asafpr** [comment]
@natir is this PR ready for merge?

Thanks

**Comment by asafpr** [comment]
> Good job @asafpr.
> 
> I have another concern this recipe has many script and binary, but many of them seem not be launch by the end-user, but only by `hydra-multi.sh`.
> 
> @bioconda/core is it ok to leave all these script and binary in $PREFIX/bin or better store them in $PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/ and change $PATH or calling path in launch script ?

Actually, most of the scripts are used by the end-user. This tool wasn't really built with usability in mind and has some steps of mixed python and bash scripts.
I understand that it will make $PREFIX/bin busy

**Comment by natir** [comment]
Ok my last concern is solved

**Review by natir — APPROVED** [review]

**Comment by natir** [comment]
@BiocondaBot please merge

**Pull request merged**